src/WorldStorage/NamespaceSerializer.cpp

```diff
@@ -131,129 +131,125 @@ namespace NamespaceSerializer
+		{ "walk_under_water_one_cm",                 Statistic::WalkUnderWaterOneCm },
 
+		// Old ones just for compatibility
+		{ "junk_fished",                             Statistic::JunkFished },
+		{ "treasure_fished",                         Statistic::TreasureFished },
 
-
+		// The old advancements
+		{ "cuberite:achievement.openInventory",      Statistic::AchOpenInventory },
+		{ "cuberite:achievement.mineWood",           Statistic::AchMineWood },
+		{ "cuberite:achievement.buildWorkBench",     Statistic::AchBuildWorkBench },
+		{ "cuberite:achievement.buildPickaxe",       Statistic::AchBuildPickaxe },
+		{ "cuberite:achievement.buildFurnace",       Statistic::AchBuildFurnace },
+		{ "cuberite:achievement.acquireIron",        Statistic::AchAcquireIron },
+		{ "cuberite:achievement.buildHoe",           Statistic::AchBuildHoe },
+		{ "cuberite:achievement.makeBread",          Statistic::AchMakeBread },
+		{ "cuberite:achievement.bakeCake",           Statistic::AchBakeCake },
+		{ "cuberite:achievement.buildBetterPickaxe", Statistic::AchBuildBetterPickaxe },
+		{ "cuberite:achievement.cookFish",           Statistic::AchCookFish },
+		{ "cuberite:achievement.onARail",            Statistic::AchOnARail },
+		{ "cuberite:achievement.buildSword",         Statistic::AchBuildSword },
+		{ "cuberite:achievement.killEnemy",          Statistic::AchKillEnemy },
+		{ "cuberite:achievement.killCow",            Statistic::AchKillCow },
+		{ "cuberite:achievement.flyPig",             Statistic::AchFlyPig },
+		{ "cuberite:achievement.snipeSkeleton",      Statistic::AchSnipeSkeleton },
+		{ "cuberite:achievement.diamonds",           Statistic::AchDiamonds },
+		{ "cuberite:achievement.portal",             Statistic::AchPortal },
+		{ "cuberite:achievement.ghast",              Statistic::AchGhast },
+		{ "cuberite:achievement.blazeRod",           Statistic::AchBlazeRod },
+		{ "cuberite:achievement.potion",             Statistic::AchPotion },
+		{ "cuberite:achievement.theEnd",             Statistic::AchTheEnd },
+		{ "cuberite:achievement.theEnd2",            Statistic::AchTheEnd2 },
+		{ "cuberite:achievement.enchantments",       Statistic::AchEnchantments },
+		{ "cuberite:achievement.overkill",           Statistic::AchOverkill },
+		{ "cuberite:achievement.bookcase",           Statistic::AchBookcase },
+		{ "cuberite:achievement.exploreAllBiomes",   Statistic::AchExploreAllBiomes },
+		{ "cuberite:achievement.spawnWither",        Statistic::AchSpawnWither },
+		{ "cuberite:achievement.killWither",         Statistic::AchKillWither },
+		{ "cuberite:achievement.fullBeacon",         Statistic::AchFullBeacon },
+		{ "cuberite:achievement.breedCow",           Statistic::AchBreedCow },
+		{ "cuberite:achievement.diamondsToYou",      Statistic::AchDiamondsToYou}
+	};
 
 	Statistic ToCustomStatistic(const std::string_view ID)
 	{
-		static const std::unordered_map<std::string_view, Statistic> CustomStatistics
```

Comment:
Checking the guard variable is going to be negligible compared to the actual container lookup time.

Reply:
well it's still not free and doesn't get us anything